src/io/perun.clj

```diff
@@ -26,6 +26,27 @@
   '[[org.pegdown/pegdown "1.6.0"]
     [circleci/clj-yaml "0.5.3"]])
 
+(deftask dump-meta
+  "Utility task to dump perun metadata via boot.util/info"
+  [r to-remove REMOVE [kw] "Keys to remove from metadata map values"]
+  (boot/with-pre-wrap fileset
+    (prn (pr-str (map #(apply dissoc % to-remove) (perun/get-meta fileset))))
+    fileset))
+
+(deftask base
```

Comment:
From the PR text:
> I added a `base` task that establishes some initial structure. I'm not yet convinced this is useful in some way. Probably tasks that use the added `:filename` key should just look that up through the fileset (and then also add it to the metadata).

I'm not sure if there's a need for this kind of task that provides an initial set of things to work with but with the present code this has been effectively handled by the markdown task based on the assumption that it comes first. (Or at least that's how I understood the code.) 

In a way the markdown task still does it so I'm not sure if it's 100% necessary. Input by you and others definitely appreciated :)